### DISCUSSION TITLE
Giving Error Warning: Invalid DOM property `stroke-width`. Did you mean `strokeWidth`?

### DISCUSSION THREAD

**Discussion by techUdayMungalpara** [tailwindlabs/heroicons#1023] [Help]
i use Heroicos in my next js project showing error in console Warning: Invalid DOM property `stroke-width`. Did you mean `strokeWidth`?
    at path
    at svg

**Comment by davidecscorp** [comment]
@techUdayMungalpara. This is a warning not an error per se, the reason you are seeing that is that you are working in a `jsx` file. Similar to how you use `className` instead of just `class`. What you have to do is change the stroke-width attribute to `strokeWidth` in the icon you are using.

**Reply by farzadso** [reply]
This is also the case when we directly import the components and modifying the JSX is not possible.

e.g.

```js
import { SpeakerWaveIcon, SpeakerXMarkIcon } from "@heroicons/react/24/outline";
```
